cmd/pomerium/main.go

```diff
@@ -213,11 +213,20 @@ func parseOptions(configFile string) (*config.Options, error) {
 	metrics.AddPolicyCountCallback(o.Services, func() int64 {
 		return int64(len(o.Policies))
 	})
-	checksumInt, err := strconv.ParseInt(fmt.Sprintf("0x%s", o.Checksum()), 0, 64)
+
+	// checksumDec, _, err := big.ParseFloat(
+	// 	fmt.Sprintf("0x%s", o.Checksum()),
+	// 	0, 64, big.ToNearestEven)
+	// if err != nil {
+	// 	log.Warn().Err(err).Msg("Could not parse config checksum into decimal")
+	// }
+	// checksumFloat64, _ := checksumDec.Float64()
+	checksumDec, _ := strconv.ParseUint(fmt.Sprintf("0x%s", o.Checksum()), 0, 64)
 	if err != nil {
-		log.Warn().Err(err).Msg("Could not parse config checksum into integer")
+		log.Warn().Err(err).Msg("Could not parse config checksum into decimal")
```

Comment:
uint :)

Reply:
Soft counter argument - my thinking was that there's really no need to get type specific outside the code itself.  We're just trying to get it into base10.  I changed a few other refs along the same lines.

Reply:
Good point.